src/foremast/awslambda/awslambda.py

```diff
@@ -61,6 +61,22 @@ def _check_lambda(self):
             pass
         return exists
 
+    def _check_lambda_alias(self):
+        """ Checks if lambda alias exists
```

Comment:
Docstring should be like:
```python
"""Check if lambda alias exists.

Returns:
...
"""
```
`pydocstyle` can help with keeping docstrings consistent.